[2025-01-31T09:58:21.308+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T09:58:21.361+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my-first-python-etl-dag.extract scheduled__2025-01-29T00:00:00+00:00 [queued]>
[2025-01-31T09:58:21.380+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my-first-python-etl-dag.extract scheduled__2025-01-29T00:00:00+00:00 [queued]>
[2025-01-31T09:58:21.381+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-31T09:58:21.416+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract> on 2025-01-29 00:00:00+00:00
[2025-01-31T09:58:21.426+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=67) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T09:58:21.429+0000] {standard_task_runner.py:72} INFO - Started process 71 to run task
[2025-01-31T09:58:21.430+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my-first-python-etl-dag', 'extract', 'scheduled__2025-01-29T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpyemmq8pt']
[2025-01-31T09:58:21.433+0000] {standard_task_runner.py:105} INFO - Job 21: Subtask extract
[2025-01-31T09:58:21.878+0000] {task_command.py:467} INFO - Running <TaskInstance: my-first-python-etl-dag.extract scheduled__2025-01-29T00:00:00+00:00 [running]> on host 0d287206d23f
[2025-01-31T09:58:22.050+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='olalekanrasaq1331@gmail.com' AIRFLOW_CTX_DAG_OWNER='Akinkunmi' AIRFLOW_CTX_DAG_ID='my-first-python-etl-dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-29T00:00:00+00:00'
[2025-01-31T09:58:22.052+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-31T09:58:22.053+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-31T09:58:22.054+0000] {logging_mixin.py:190} INFO - Current task name:extract state:running start_date:2025-01-31 09:58:21.362172+00:00
[2025-01-31T09:58:22.055+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag and current dag run status:running
[2025-01-31T09:58:22.056+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T09:58:22.058+0000] {logging_mixin.py:190} INFO - Inside Extract
[2025-01-31T09:58:22.060+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-31T09:58:22.083+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T09:58:22.085+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my-first-python-etl-dag, task_id=extract, run_id=scheduled__2025-01-29T00:00:00+00:00, execution_date=20250129T000000, start_date=20250131T095821, end_date=20250131T095822
[2025-01-31T09:58:22.105+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-31T09:58:22.106+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-31T09:58:22.107+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag queued_at:2025-01-31 09:58:15.100369+00:00
[2025-01-31T09:58:22.108+0000] {logging_mixin.py:190} INFO - Task hostname:0d287206d23f operator:PythonOperator
[2025-01-31T09:58:22.168+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T09:58:22.474+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-31T09:58:22.477+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
